skyvern/services/script_service.py

```
@@ -1485,6 +1488,36 @@ async def extract(
         return block_result.output_parameter_value
 
 
+async def validate(
+    complete_criterion: str | None = None,
+    terminate_criterion: str | None = None,
+    error_code_mapping: dict[str, str] | None = None,
+    label: str | None = None,
+) -> None:
+    """Validate function that behaves like a ValidationBlock"""
+    if not complete_criterion and not terminate_criterion:
+        raise Exception("Both complete criterion and terminate criterion are empty")
```

Ruff (0.13.2)

Create your own exception

(TRY002)